### DISCUSSION TITLE
Alipay please, I think you can get some fund from chinese developer.

### DISCUSSION THREAD

**Discussion by adoin** [zloirock/core-js#1198] [Q&A]
Please provide Alipay payment qrcode.

**Comment by fangbiexiansheng** [comment]
+1

**Reply by Rudxain** [reply]
I have no idea why you posted this. You can give a "thumbs up" reaction, if you want your upvotes to be publicly associated to your account

**Reply by adoin** [reply]
> I have no idea why you posted this. You can give a "thumbs up" reaction, if you want your upvotes to be publicly associated to your account

It's hard to understand different language culture, especially Internet-related.

**Reply by Rudxain** [reply]
> It's hard to understand different language culture, especially Internet-related.

I guess I also don't understand 😅. How is that related to language?

**Reply by adoin** [reply]
> > It's hard to understand different language culture, especially Internet-related.
> 
> I guess I also don't understand 😅. How is that related to language?

Internet language culture, hard to explain, i.e. you say lmao to describe funny we don't; we say NB to describe powerfull or awesome you don't. Here +1 just means 'he said what I want to say', and 1 to is yes or yep or ok, while +1s means pray for a greatest leader Jiang, even he died mouths before.

**Reply by Rudxain** [reply]
> Internet language culture, hard to explain, i.e. you say lmao to describe funny we don't; we say NB to describe powerfull or awesome you don't. Here +1 just means 'he said what I want to say', and 1 to is yes or yep or ok

I already knew that, but I think I get your point now. I also used to say "+1" "back in the day" (I still do!). But I don't find it useful if there are 2 buttons that do the same job ("upvote" and "like").

Anyways, thanks for explaining.

> while +1s means pray for a greatest leader Jiang, even he died mouths before.

what? do you mean "Jiāng Zémín"?

**Reply by zha-ji-tui** [reply]
> > Internet language culture, hard to explain, i.e. you say lmao to describe funny we don't; we say NB to describe powerfull or awesome you don't. Here +1 just means 'he said what I want to say', and 1 to is yes or yep or ok
> 
> I already knew that, but I think I get your point now. I also used to say "+1" "back in the day" (I still do!). But I don't find it useful if there are 2 buttons that do the same job ("upvote" and "like").
> 
> Anyways, thanks for explaining.
> 
> > while +1s means pray for a greatest leader Jiang, even he died mouths before.
> 
> what? do you mean "Jiāng Zémín"?

yes

**Comment by zloirock** [comment]
I've added Alipay:

![**Alipay**](https://user-images.githubusercontent.com/2213682/219464783-c17ad329-17ce-4795-82a7-f609493345ed.png)

**Reply by adoin** [reply]
Put this on top dude, page footer or header, so that people can help you.

**Reply by hutaod** [reply]
Personal support is 100 RMB, come on!

**Reply by zha-ji-tui** [reply]
> I've added Alipay:
> 
> ![Alipay](https://user-images.githubusercontent.com/2213682/219464783-c17ad329-17ce-4795-82a7-f609493345ed.png)

thanks form China

**Reply by aijun198600** [reply]
thanks for core-js jobs, personal support 150 RMB, hope every thing will be better!

**Reply by zha-ji-tui** [reply]
I supported ￥50

**Reply by xudatou-94** [reply]
supported ￥50

**Reply by xsjohn0306** [reply]
supported ￥100

**Reply by Beelkic** [reply]
supported ¥100

**Reply by aaurix** [reply]
supported ¥100

**Reply by hotdogc1017** [reply]
supported ￥20
